packages/main/src/plugin/util/manifest.ts

```diff
@@ -0,0 +1,52 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ImageInfo } from '../api/image-info.js';
+
+// 50 KB
```

Comment:
looks like images with tons of arches like docker.io has only 5kb manifest so size should be good

just a small comment, usually the comment will be sooner or later out of sync with the value

so maybe, use value instead of 51200 and as comment 50 KB

something like
```
const KB = 1000;
const GUESSED_MANIFEST_SIZE = 50 * KB;
```